test/devnet-mint.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai";
+import { starknet } from "hardhat";
+
+import { TIMEOUT } from "./constants";
+
+describe("Devnet mint", function () {
+    this.timeout(TIMEOUT);
+    let balance = 0,
+        address = "";
+    const div = 1e16; // To round off large numbers
+    const diff = 25 * div;
+
+    before(async () => {
+        const accounts = await starknet.devnet.getPredeployedAccounts();
```

Comment:
It doesn't have to be a predeployed account. The address being funded doesn't even have to be that of a deployed account (makes sense if you are deploying a new account - the address needs to be funded even though the account is not yet on the network).

Therefore a simple dummy address should work (and probably no need for the `before` block)

Reply:
Aah, aye cool! Thanks!